deploy/helm/charts/templates/deployment.yaml

```diff
@@ -87,6 +87,10 @@ spec:
             - name: OPENEBS_IO_NFS_SERVER_NS
               value: {{ .Values.nfsProvisioner.nfsServerNamespace }}
             {{- end }}
+            {{- if .Values.nfsServer.imagePullSecret }}
```

Comment:
Helm chart uses 0.7.1 release and changes to support this parameter will be available in *ci* image only, not in in 0.7.1 release. Can you move these helm changes to separate PR or create an issue to add these changes once changes are available in the release? Ref:https://github.com/openebs/dynamic-nfs-provisioner/issues/88